Yafc/Windows/NeverEnoughItemsPanel.cs

```diff
@@ -186,7 +187,9 @@ private void DrawRecipeEntry(ImGui gui, RecipeEntry entry, bool production) {
                 float bh = CostAnalysis.GetBuildingHours(recipe, entry.recipeFlow);
                 if (bh > 20) {
                     gui.BuildText(DataUtils.FormatAmount(bh, UnitOfMeasure.None, suffix: "bh"), TextBlockDisplayStyle.Centered);
-                    _ = gui.BuildButton(gui.lastRect, SchemeColor.None, SchemeColor.Grey).WithTooltip(gui, "Building-hours.\nAmount of building-hours required for all researches assuming crafting speed of 1");
+
+                    string buildingHoursMessage = "Building-hours.\nAmount of building-hours required for all researches assuming crafting speed of 1";
+                    _ = gui.BuildButton(gui.lastRect, SchemeColor.None, SchemeColor.Grey).WithTooltip(gui, buildingHoursMessage);
```

Comment:
I'd prefer this. To me, starting the line with `.WithTooltip` makes it easier to understand what the string does. (And then immediately ignore the string)

```suggestion
                    _ = gui.BuildButton(gui.lastRect, SchemeColor.None, SchemeColor.Grey)
                        .WithTooltip(gui, "Building-hours.\nAmount of building-hours required for all researches assuming crafting speed of 1");
```

Reply:
Good idea. I'll update the PR to use this approach instead of a separate string.